src/lib/loaders/getCanonicalResourceDirectiveField.ts

```diff
@@ -0,0 +1,24 @@
+import { FieldNode, visit, BREAK } from "graphql"
+
+export const getCanonicalResourceDirectiveForField = (
+  fieldNode: FieldNode
```

Comment:
We’re not actually passing in a field node, are we?

Reply:
Ah yea, it's a `DocumentNode`.